inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/project/KnowledgeBaseListPanel.java

```diff
@@ -57,7 +59,9 @@ public KnowledgeBaseListPanel(String id, IModel<Project> aProjectModel,
         overviewList = new OverviewListChoice<>("knowledgebases");
         overviewList.setChoiceRenderer(new ChoiceRenderer<>("name"));
         overviewList.setChoices(
-                LambdaModel.of(() -> kbService.getKnowledgeBases(projectModel.getObject())));
+                LambdaModel.of(() -> kbService.getKnowledgeBases(projectModel.getObject()).stream()
```

Comment:
Is there a reason why you chose `stream` over `Collections::sort`?

Reply:
less code and looks nicer